.pre-commit-config.yaml

```diff
@@ -39,20 +39,12 @@ repos:
     hooks:
       - id: codespell
         exclude_types: [csv, json]
-  - repo: https://github.com/PyCQA/flake8
-    rev: 6.0.0
+  - repo: https://github.com/charliermarsh/ruff-pre-commit
```

Comment:
You want to run this before isort & black.

If it fixes thing, it will be formatted after that...

Reply:
Ok thanks, will put it in a separate PR, I just merged this one :)

Reply:
https://github.com/plugwise/python-plugwise/pull/282